apps/backend/src/main.py

```
@@ -71,6 +76,13 @@ async def lifespan(app: FastAPI):
         await init_database()
         logger.info("Database initialized successfully")
         
+        # TODO: Temporarily disable polling service due to database session management issues
+        # This will be re-enabled after refactoring the PollingService to properly manage sessions
+        # polling_service = PollingService()
+        # app.state.polling_service = polling_service
+        # await polling_service.start_polling()
+        logger.info("Polling service temporarily disabled during refactoring")
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)